addons/website/views/snippets/s_numbers_boxed.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_boxed" name="Numbers boxed">
+    <section class="s_numbers_boxed pt80 pb80 o_cc o_cc5">
+        <div class="container">
+            <div class="row o_grid_mode" style="column-gap: 48px;" data-row-count="11">
+                <div class="o_grid_item g-height-2 g-col-lg-12 col-12 col-lg-12" style="grid-area: 1 / 1 / 3 / 13;">
+                    <h2 style="text-align: center;">Insights, stats, and metrics</h2>
+                    <p style="text-align: center;">This section allows to highlight key statistics.<br/><br/></p>
+                </div>
+                <div class="o_grid_item g-height-5 g-col-lg-4 col-6 col-lg-3 offset-lg-1 pt48 pb48 border rounded text-center" style="grid-area: 4 / 1 / 9 / 5; --grid-item-padding-y: 72px; border-radius: 9.6px !important;">
+                    <h3 class="display-3">12k</h3>
+                    <p class="bold">Useful options</p>
+                </div>
+                <div class="o_grid_item g-height-5 g-col-lg-4 col-6 col-lg-3 offset-lg-1 pt48 pb48 border rounded text-center" style="grid-area: 4 / 5 / 9 / 9; --grid-item-padding-y: 72px; border-radius: 9.6px !important;">
+                    <h3 class="display-3">45%</h3>
+                    <p class="fw-bold">More leads</p>
+                </div>
+                <div class="o_grid_item g-height-5 g-col-lg-4 col-6 col-lg-3 offset-lg-1 pt48 pb48 border rounded text-center" style="grid-area: 4 / 9 / 9 / 13; --grid-item-padding-y: 72px; border-radius: 9.6px !important;">
```

Comment:
Ops 😉

```suggestion
                <div class="o_grid_item g-height-5 g-col-lg-4 col-6 col-lg-3 offset-lg-1 pt48 pb48 border rounded text-center" style="grid-area: 4 / 9 / 9 / 13; --grid-item-padding-y: 72px;">
```

Reply:
Don't we want the items to be rounded? Unless I'm missing something 😅 Would you like to remove the roundness to fix the visual result on mobile ?

Reply:
> Unless I'm missing something 😅

you're missing the `rounded`class  😉

Reply:
Actually had a [comment](https://github.com/odoo/odoo/pull/175509#discussion_r1705538862) about that on the `s_cta_badge` so I didn't use the classes anymore since then 😬

Apparently we should write our style as the web_editor would apply the style, so I got requested to switch to inline-style 😅

Reply:
Ops.. I've misread `9` to`6` , my bad 🤭.
I was the one missing something apparently 🤓